clang/lib/CodeGen/CodeGenAction.cpp

```diff
@@ -1061,6 +1070,16 @@ CodeGenAction::CreateASTConsumer(CompilerInstance &CI, StringRef InFile) {
     CI.getPreprocessor().addPPCallbacks(std::move(Callbacks));
   }
 
+  if (CI.getFrontendOpts().GenReducedBMI &&
+      !CI.getFrontendOpts().ModuleOutputPath.empty()) {
+    std::vector<std::unique_ptr<ASTConsumer>> Consumers{2};
```

Comment:
Minor nit: https://llvm.org/docs/CodingStandards.html#id28
```suggestion
    std::vector<std::unique_ptr<ASTConsumer>> Consumers(2);
```